apps/site/pages/en/learn/typescript/transpile.md

```diff
@@ -88,31 +89,4 @@ const justine: User = {
 const isJustineAnAdult: string = isAdult(justine, "I shouldn't be here!");
 ```
 
-And this is what TypeScript has to say about this:
```

Comment:
This is kind of useful because it's the output that the reader/user will see in this terminal